src/Microsoft.PowerShell.Commands.Management/commands/management/WMIHelper.cs

```diff
@@ -1532,10 +1532,8 @@ private void HandleChildJobStateChanged(object sender, JobStateEventArgs e)
             if (e.JobStateInfo.State == JobState.Blocked)
             {
                 // increment count of blocked child jobs
-                lock (_syncObject)
```

Comment:
I just found that we don't even compile the `WMIHelper.cs` (see `Microsoft.PowerShell.Commands.Management.csproj` for the `<compile remove=` entries). So, please revert changes in this file.